sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/ModelsLifecycleSyncSamples.java

```diff
@@ -45,11 +47,16 @@ public static void main(String[] args) throws IOException, InterruptedException
         runModelLifecycleSample();
     }
 
+    public static Function<Integer, String> nameGenerator = (maxLength) -> {
```

Comment:
I had to factor out some of the unique Id generation logic to the samples and e2e tests since the e2e tests need to record the random numbers that we generate.